packages/nimble-components/src/table/styles.ts

```diff
@@ -117,6 +118,25 @@ export const styles = css`
         position: absolute;
     }
 
+    .column-divider:hover {
+        border-color: ${borderHoverColor};
+    }
+
+    .column-divider.visible {
+        display: block;
+        z-index: ${ZIndexLevels.zIndex1};
```

Comment:
Maybe `z-index` should just go in the base `.column-divider` styling, so it doesn't have to be set in two places.